at_onboarding_flutter/example/lib/main.dart

```diff
@@ -37,36 +37,36 @@ class _MyAppState extends State<MyApp> {
         ),
         body: Builder(
           builder: (BuildContext context) => Center(
-              child: Column(
-                  children:<Widget>[
-                        TextButton(
-                onPressed: () async {
-                  Onboarding(
-                      context: context,
-                      atClientPreference: atClientPrefernce,
-                      domain: AppConstants.rootDomain,
-                      appColor: const Color.fromARGB(255, 240, 94, 62),
-                      onboard: (Map<String?, AtClientService> value, String? atsign) {
-                        AtService.getInstance().atClientServiceMap = value;
-                        _logger.finer('Successfully onboarded $atsign');
-                      },
-                      onError: (Object? error) {
-                        _logger.severe('Onboarding throws $error error');
-                      },
-                      nextScreen: DashBoard(),
-                      appAPIKey: AppConstants.devAPIKey);
-                },
-                child: const Text(AppStrings.scan_qr)),
-                        CustomResetButton(
-                          loading: false,
-                          buttonText: "Reset",
-                          width: 90,
-                          height: 40,
-                        ),
-                  ],
-              ),
+            child: Column(
+              children: <Widget>[
+                TextButton(
+                    onPressed: () async {
+                      Onboarding(
+                        context: context,
+                        atClientPreference: atClientPrefernce,
+                        appColor: const Color.fromARGB(255, 240, 94, 62),
+                        onboard: (Map<String?, AtClientService> value, String? atsign) {
+                          AtService.getInstance().atClientServiceMap = value;
+                          _logger.finer('Successfully onboarded $atsign');
+                        },
+                        onError: (Object? error) {
+                          _logger.severe('Onboarding throws $error error');
+                        },
+                        rootEnvironment: RootEnvironment.Production,
```

Comment:
If you are using RootEnvironment.Production as example then APIKey has to be provided.

Reply:
oops i forgot lemme add it to `staging`